src/components/ui/input-validation-error/input-validation-error.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+type Props = {
```

Comment:
can u explain why do u use type instead of interface? and differences between these?

Reply:
you are right, and here for object u are using `type`

Reply:
ok ill fix it everywhere later